assets/src/hooks/v2/use_interval_paging.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect, useRef } from "react";
+
+interface UseIntervalPagingProps {
+  numPages: number; // Total number of pages to cycle through
+  cycleIntervalMs: number; // In milliseconds, the interval for cycling pagesß
```

Comment:
`ß`?

Reply:
whoops that's me misclicking a command-S to save